Calibration/IsolatedParticles/plugins/IsolatedTracksCone.cc

```diff
@@ -1,96 +1,594 @@
+  delete t_v_cone_hnNearTRKs; 
+  delete t_v_cone_hnLayers_maxNearP; 
+  delete t_v_cone_htrkQual_maxNearP; 
+  delete t_v_cone_hmaxNearP_goodTrk;
+  delete t_v_cone_hmaxNearP;    
+  delete t_trkNOuterHits;
+  delete t_trkNLayersCrossed;
+  delete t_dtFromLeadJet;
+  delete t_trkP;
+  delete t_simP;
+  delete t_trkPt;
+  delete t_trkEta;
+  delete t_trkPhi;
+  delete t_e3x3;
+  delete t_v_eDR;
+  delete t_v_eMipDR;
+  delete t_h3x3;
+  delete t_h5x5;
+  delete t_hsim3x3;
+  delete t_hsim5x5;
+  delete t_nRH_h3x3;
+  delete t_nRH_h5x5;
+  delete t_hsim3x3Matched;
+  delete t_hsim5x5Matched;
+  delete t_hsim3x3Rest;
+  delete t_hsim5x5Rest;
+  delete t_hsim3x3Photon;
+  delete t_hsim5x5Photon;
+  delete t_hsim3x3NeutHad;
+  delete t_hsim5x5NeutHad;
+  delete t_hsim3x3CharHad;
+  delete t_hsim5x5CharHad;
+  delete t_hsim3x3PdgMatched;
+  delete t_hsim5x5PdgMatched;
+  delete t_hsim3x3Total;
+  delete t_hsim5x5Total;
+  delete t_hsim3x3NMatched;
+  delete t_hsim3x3NRest;
+  delete t_hsim3x3NPhoton;
+  delete t_hsim3x3NNeutHad;
+  delete t_hsim3x3NCharHad;
+  delete t_hsim3x3NTotal;
+  delete t_hsim5x5NMatched;
+  delete t_hsim5x5NRest;
+  delete t_hsim5x5NPhoton;
+  delete t_hsim5x5NNeutHad;
+  delete t_hsim5x5NCharHad;
+  delete t_hsim5x5NTotal;
+  delete t_distFromHotCell_h3x3;
+  delete t_ietaFromHotCell_h3x3;
+  delete t_iphiFromHotCell_h3x3;
+  delete t_distFromHotCell_h5x5;
+  delete t_ietaFromHotCell_h5x5;
+  delete t_iphiFromHotCell_h5x5;
+  delete t_trkHcalEne;
+  delete t_trkEcalEne;
+  delete t_v_hsimInfoConeMatched;
+  delete t_v_hsimInfoConeRest;
+  delete t_v_hsimInfoConePhoton;
+  delete t_v_hsimInfoConeNeutHad;
+  delete t_v_hsimInfoConeCharHad;
+  delete t_v_hsimInfoConePdgMatched;
+  delete t_v_hsimInfoConeTotal;
+  delete t_v_hsimInfoConeNMatched;
+  delete t_v_hsimInfoConeNRest;
+  delete t_v_hsimInfoConeNPhoton;
+  delete t_v_hsimInfoConeNNeutHad;
+  delete t_v_hsimInfoConeNCharHad;
+  delete t_v_hsimInfoConeNTotal;
+  delete t_v_hsimCone;
+  delete t_v_hCone;
+  delete t_v_nRecHitsCone;
+  delete t_v_nSimHitsCone;
+  delete t_v_distFromHotCell;
+  delete t_v_ietaFromHotCell;
+  delete t_v_iphiFromHotCell;
+  delete t_v_hlTriggers;
+  delete t_hltHB;
+  delete t_hltHE;
+  delete t_hltL1Jet15;
+  delete t_hltJet30;
+  delete t_hltJet50;
+  delete t_hltJet80;
+  delete t_hltJet110;
+  delete t_hltJet140;
+  delete t_hltJet180;
+  delete t_hltL1SingleEG5;
+  delete t_hltZeroBias;
+  delete t_hltMinBiasHcal;
+  delete t_hltMinBiasEcal;
+  delete t_hltMinBiasPixel;
+  delete t_hltSingleIsoTau30_Trk5;
+  delete t_hltDoubleLooseIsoTau15_Trk5;
+  delete t_v_RH_h3x3_ieta;
+  delete t_v_RH_h3x3_iphi;
+  delete t_v_RH_h3x3_ene ;
+  delete t_v_RH_h5x5_ieta;
+  delete t_v_RH_h5x5_iphi;
+  delete t_v_RH_h5x5_ene ;
+  delete t_v_RH_r26_ieta ;
+  delete t_v_RH_r26_iphi ;
+  delete t_v_RH_r26_ene  ;
+  delete t_v_RH_r44_ieta ;
+  delete t_v_RH_r44_iphi ;
+  delete t_v_RH_r44_ene  ;
+  delete t_irun;
+  delete t_ievt;
+  delete t_ilum;
```

Comment:
If I am not mistaken, these could all become `unique_ptr`s (with the appropriate changes everywhere)